Source/WebCore/css/query/MediaQueryFeatures.cpp

```diff
@@ -266,12 +266,8 @@ const FeatureSchema& colorGamut()
         [](auto& context) {
             // FIXME: At some point we should start detecting displays that support more colors.
             MatchingIdentifiers identifiers { CSSValueSRGB };
-#if HAVE(IOSURFACE_RGB10)
             if (screenContentsFormat(context.document->protectedFrame()->mainFrame().protectedVirtualView().get()) == ContentsFormat::RGBA10)
```

Comment:
I think [284617@main](https://commits.webkit.org/284617@main) conflated "wants extended color" and the `screenContentsFormat`. It doesn't really make sense to ask "what pixel format does the screen use", so I don't think we should be using `ContentsFormat` to communicate the screen's capabilities.

Reply:
I restored this function to be as it was before [284617@main](https://commits.webkit.org/284617@main). I made `screenContentsFormat` use `ScreenData` not vice versa.